### PR TITLE
Use CommonModelGenerator in commons tests

### PR DESCRIPTION
**Pull request by alulc**

### CODE REVIEW

File: src/test/java/marquez/common/models/CommonModelGenerator.java [hunk 1]
```diff
@@ -92,8 +92,12 @@ public static DbName newDbNameWith(String value) {
   }
 
   public static ConnectionUrl newConnectionUrl() {
+    return newConnectionUrlWithDatasourceType(newDatasourceType());
+  }
+
+  public static ConnectionUrl newConnectionUrlWithDatasourceType(DatasourceType datasourceType) {
```
Comment (wslulciuc) [inline src/test/java/marquez/common/models/CommonModelGenerator.java]: _Minor:_ The naming convention we have for generators is `newClazzWith(Overrides...)`, where `Clazz` is the class you'd like to instantiate and `Overrides...` are any overrides you'd like to provide.

So a generator method for `ConnectionUrl` that overrides the `DatasourceType` would be:

```java
newConnectionUrlWith(DatasourceType datasourceType)
```

File: src/test/java/marquez/common/models/ConnectionUrlTest.java [hunk 2]
```diff
@@ -22,7 +22,7 @@
 
 @Category(UnitTests.class)
 public class ConnectionUrlTest {
-  private static final DatasourceType DATASOURCE_TYPE = DatasourceType.POSTGRESQL;
+  private static final DatasourceType DATASOURCE_TYPE = CommonModelGenerator.newDatasourceType();
```
Comment (wslulciuc) [inline src/test/java/marquez/common/models/ConnectionUrlTest.java]: _Minor_: I would statically import `newDatasourceType()`

```java
import static marquez.common.models.CommonModelGenerator.newDatasourceType;

private static final DatasourceType DATASOURCE_TYPE = newDatasourceType()
```

File: src/test/java/marquez/common/models/CommonModelGenerator.java (hunk 1 above)
Comment (alulc) [reply src/test/java/marquez/common/models/CommonModelGenerator.java]: Ok.

File: src/test/java/marquez/common/models/ConnectionUrlTest.java (hunk 2 above)
Comment (alulc) [reply src/test/java/marquez/common/models/ConnectionUrlTest.java]: Will do.

**Pull request merged**